packages/checkbox/package.json

```diff
@@ -12,8 +12,10 @@
     "access": "public"
   },
   "dependencies": {
-    "@leafygreen-ui/lib": "^5.1.1",
-    "@leafygreen-ui/theme": "^2.0.1"
+    "@leafygreen-ui/lib": "^5.1.1"
+  },
+  "devDependencies": {
+    "@leafygreen-ui/palette": "^2.0.2"
```

Comment:
Why is this change being made?

Reply:
Isn't deprecating theme all together a goal? This is the last component that was relying on theme, so once we remove it here then we're done with the package throughout our repo. Further, the theme was only used in three places so I just replaced the reference to theme with the same hex value, with the hope that we'd move over to the palette entirely with this component eventually.

Reply:
Whoops, thought this was moving theme from a dependency to a devDependency. Why is this a devDependency I suppose is a better question 😅

Reply:
It's only being used in Storybook, for the three styles I changed, I inlined the hex values (so they're staying the same as they are)